config/settings.defaults.coffee

```diff
@@ -25,44 +25,39 @@ module.exports =
 			host: process.env['PUBSUB_REDIS_HOST'] or process.env['REDIS_HOST'] or "localhost"
 			port: process.env['PUBSUB_REDIS_PORT'] or process.env['REDIS_PORT'] or "6379"
 			password: process.env["PUBSUB_REDIS_PASSWORD"] or process.env["REDIS_PASSWORD"] or ""
-			redisOptions:	
-				keepAlive: 100
-				
+			maxRetriesPerRequest: 0
```

Comment:
shall we have the default at 20 with an env `MAX_RETRIES_PER_REQUEST` passed through to set this?